pkg/apitype/policy.go

```diff
@@ -165,3 +199,10 @@ type PolicyGroupSummary struct {
 	NumStacks             int    `json:"numStacks"`
 	NumEnabledPolicyPacks int    `json:"numEnabledPolicyPacks"`
 }
+
+// GetPolicyPackConfigSchemaResponse is the response that includes the JSON
+// schemas of Policies within a particular Policy Pack.
+type GetPolicyPackConfigSchemaResponse struct {
+	// The JSON schema for each Policy's configuration.
+	ConfigSchema map[string]*PolicyConfigSchema `json:"configSchema,omitempty"`
```

Comment:
I don't think we need to use a pointer for the value. If the policy doesn't have a schema, it won't be included in the map.

```suggestion
	ConfigSchema map[string]PolicyConfigSchema `json:"configSchema,omitempty"`
```